applications/matter_bridge/Kconfig.sysbuild

```diff
@@ -52,6 +64,12 @@ config MCUBOOT_UPDATEABLE_IMAGES
 
 endif
 
+# Use common SUIT envelope templates prepared for Matter purpose
+# The templates are located in the samples/matter/common/suit_templates directory.
+# To use custom templates set this config to n
+config MATTER_USE_COMMON_SUIT_ENVELOPE_TEMPLATES
```

Comment:
I think that the convention on the Arek's PR changed again, sorry for additional work with alignment several times. It seems that this part should be removed from the Kconfig,sysbuild and you should add the following line to the boards/nrf54h20dk_nrf54h20_cpuapp.conf:

CONFIG_SUIT_ENVELOPE_TEMPLATE="${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common/suit_templates/app_envelope.yaml.jinja2"

Please @ArekBalysNordic could you double check if this is correct?